docs/integrations/integration_datahub_template.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Integrating DataHub With Great Expectations
+authors:
+    name: AUTHOR NAME
+    url: https://datahubproject.io/
+---
+
+:::info
+* Maintained By: DataHub
+* Status: Beta
+* Support/Contact: https://slack.datahubproject.io/
+:::
+
+### Introduction
+This integration allows you to push the results of running expectations into DataHub (https://datahubproject.io/). DataHub is a metadata platform which enables search & discovery, federated governance, and data observability for the Modern Data Stack.
+
+
+### Technical background
+We have implemented [Custom Action](https://docs.greatexpectations.io/docs/terms/action#how-to-create) named `DataHubValidationAction`
```

Comment:
Instead of "we have implemented" 

Use "There is a custom action named `DataHubValidationAction` which allows you to view expectation results inside of DataHub"

Reply:
Done